app/templates/partials/questions/repeatinganswer.html

```diff
@@ -27,7 +29,9 @@ <h3 class="neptune">{{ _('Person ') }}<span class="js-household-loopindex">{{ lo
         <fieldset class="question__fieldset">
           <legend class="question__legend u-vh">{{ _('Person ') }} {{ loop.index }}</legend>
           {% for answer in composite_answer %}
-            {% include theme('partials/answer.html') %}
+            {% with render_guidance = False %}
```

Comment:
Do we actually need this boolean flag as it's not reassigned and it's only used in one place?